index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ ] **Project Requirement** 

> -  A .html file that includes a header with the text "Hello Microverse!". Use the h1 tag for that.  

:neckbeard: According to project requirements, you should add your `h1` element inside the `header` semantic tag. I suggest always using semantic tags in your HTML projects as semantics tags add accessibility 💯 to your app.